app/jenkins-for-jira-ui/src/components/JenkinsConfigurationForm/JenkinsConfigurationForm.tsx

```diff
@@ -56,17 +62,38 @@ const JenkinsConfigurationForm = ({
 	setHasError,
 	isLoading
 }: JenkinsConfigurationFormProps) => {
+	const analyticsClient = new AnalyticsClient();
```

Comment:
Not sending analytics inside useEffect in this component so fine to define here. Thoughts on moving outside the component though just for consistencies sake?